ambari/service_checks.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "agent_version": "6.12.0",
+        "integration":"ambari",
+        "groups": [],
```

Comment:
dont know what to put in here

Reply:
In here should be the name of the tags you send alongside your service check. So for the `ambari.can_connect` check, that would be `"groups": ["host", "url"]`